Client/Frontend/Browser/TabManager.swift

```diff
@@ -39,7 +39,21 @@ class TabManager {
         return tabs[index]
     }
 
+    func getTab(webView: WKWebView) -> Browser? {
+        
+        for i in 0..<count {
+            if tabs[i].webView === webView {
+                return tabs[i]
+            }
+        }
+        return nil
+    }
+
     func selectTab(tab: Browser?) {
+        
+        if tab == nil {
+            return
+        }
```

Comment:
That's cool, I found the nil switching to a blank view not a major issue but figured it would be appropriate to add the early return on nil, since I was creating a potential nil return in my getTab(webView) method. I'll have this removed. :)